src/lntors/mod.rs

```diff
@@ -5,6 +5,8 @@ mod function;
 mod typen;
 
 pub fn lntors(entry_file: String) -> Result<String, Box<dyn std::error::Error>> {
+    // TODO: Figure out a better way to include custom Rust functions that we may then bind
+    let preamble = include_str!("../std/root.rs").to_string();
```

Comment:
This one is a big TODO so I want to think a bit about options here. It won't block this PR, but it will block the first non-root-scope standard library.